src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -22,6 +19,17 @@ import environment from 'platform/utilities/environment';
 const vaGovFullDomain = environment.BASE_URL;
 const logoSrc = `${vaGovFullDomain}/img/design/logo/va-logo.png`;
 
+const AlertBox = loadable(() =>
+  import(/* webpackChunkName: "signInModal" */ '@department-of-veterans-affairs/component-library/AlertBox'),
+);
+const Modal = loadable(() =>
+  import(/* webpackChunkName: "signInModal" */ '@department-of-veterans-affairs/component-library/Modal'),
+);
+
+const Telephone = loadable(() =>
+  import(/* webpackChunkName: "signInModal" */ '@department-of-veterans-affairs/component-library/Telephone'),
+);
```

Comment:
These design system components are likely being used by all of our application code too, so I feel we don't need to lazy load them?

Reply:
@ncksllvn I removed any shared modules and instead just lazy loaded non-shared components of Identity.  Sorry this took so long there were a lot of issues with unit-tests & e2e testing in cypress that kept breaking 🤷 .